src/content/updates/release-information/highlights.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Highlights
+tabs: ['Highlights', "Changelog", 'Known issues']
+---
+
+## Exciting changes
+
+The Carbon team is excited to announce the release of Carbon v10, a major update to IBM's open-source design system.
+
+### New look for Carbon
+
+The new [IBM Design Language](https://www.ibm.com/design/language/) represents a complete overhaul of IBM’s design ethos. Designed from the ground up as a unified, flexible, and expressive set of principles, guidelines, and assets, the new design language redefines the essence of “IBM Design.”
+
+The changes to the Carbon Design System reflect the new IBM Design Language. Carbon is the applied usage of the IBM Design Language for web and software products and offerings.
+
+### Redesigned website
+
+With an all-new look and feel based on the new IBM Design Language standards, the Carbon website is better than ever. The site features improved information architecture, a new Help section, and faster performance, ensuring that users can find exactly what they need in record time.
```

Comment:
Yes, it should be a hyperlink. And, no (though this content was pre-existing), I don't think we should capitalize that page name.

Also, I'd suggested "Release information" (because it was previously "Release highlights" but "highlights" is now just a single page)